andela/AndelaEatsUI
# #160551202 Implement add menu functionality

**What does the PR do?**
- This PR Implements `Add Menu Functionality`

**Description of Task to be completed?**
- add menu functionality
- update adminMenuReducer with create menu action
- add create menu actions
- write test for added actions

**How should this be manually tested?**
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git)
-  cd into AndelaEatsUI and checkout to this branch `ft-implement-add-menu-functionality-160551202`
- run these commands `npm run bundle` and `npm run dev` 
- Log in as an `Admin`, navigate to `admin dashboard`, select `Menu Icon`, and click on `Add Menu`
- run the command `npm run test-dev` to ensure all tests are passing

**What are the relevant pivotal tracker stories?**
- [#160551202](https://www.pivotaltracker.com/story/show/160551202)

**Screenshots**
- N/A
